src/main/java/dev/codesoapbox/dummy4j/AbstractDummy4jBuilder.java

```diff
@@ -46,16 +46,12 @@ public T locale(String... locale) {
     }
 
     private List<String> varArgsToList(String... elements) {
-        if (theOnlyElementIsNull(elements)) {
+        if (elements.length == 1 && elements[0] == null) {
```

Comment:
What if every element in varrargs is null?

Reply:
Same as with `Arrays.asList(null, null, null);`, so probably not our problem.